AndroidSDKGcm/src/main/AndroidManifest.xml

```diff
@@ -2,12 +2,13 @@
           xmlns:android="http://schemas.android.com/apk/res/android">
 
     <uses-sdk android:minSdkVersion="14"/>
-    <!-- Permissions for GCM -->
+    <!-- Permissions for GCM. -->
     <uses-permission android:name="com.google.android.c2dm.permission.RECEIVE"/>
     <permission
         android:name="${applicationId}.permission.C2D_MESSAGE"
         android:protectionLevel="signature"/>
     <uses-permission android:name="${applicationId}.permission.C2D_MESSAGE"/>
+
```

Comment:
empty line

Reply:
That was planned empty line. It will separate permissions and application.